include/psp2/promoterutil.h

```diff
@@ -19,6 +19,15 @@ typedef struct ScePromoterUtilityLAUpdate {
 	char path[128];    //!< Directory of extracted LA update data.
 } ScePromoterUtilityLAUpdate;
 
+/** Parameters for scePromoterUtilityPromoteImport() */
+typedef struct ScePromoterUtilImportParams{
+	char path[0x80]; //!< Install path usually (ux0:/temp/game) 
+	char titleid[0xC]; //!< Game titleid
+	uint32_t type; //!< Package Type (0x3 on PSM, 0x1 on VITA)
+	uint32_t attribute; //!< Appears to be 0x1 on PSM content but 0x00 on Vita.
```

Comment:
maybe you should make an enum for these since you know some of the values

Reply:
type could also be 0=PSP I guess, see PKG on ps3 dev wiki

Reply:
Pretty sure content manager uses regular PromotePkgWithRif on PSP Content ...
but thats just speculation based on how CM acts when copying PSP stuff.
i have not looked into it much furher than what i allready found.